include/wx/defs.h

```diff
@@ -1736,6 +1736,19 @@ wxALLOW_COMBINING_ENUMS(wxDirection, wxGeometryCentre)
 #define wxLI_HORIZONTAL         wxHORIZONTAL
 #define wxLI_VERTICAL           wxVERTICAL
 
+/*
+    wxTextCtrl flags also used by other controls.
+
+    wxComboBox, wxSpinCtrl and may be others can also use some of these flags,
```

Comment:
```suggestion
    wxComboBox, wxSpinCtrl and maybe others can also use some of these flags,
```

Sorry for the nitpick.

Reply:
Fixed, thanks!